.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -4,4 +4,18 @@ about: Do you need support?
 
 ---
 
+## Habitat-Sim version
+vx.x.x
+
+Habitat is under active development, and we advise users to restrict themselves to stable releases. Are you using the latest [release version](https://github.com/facebookresearch/habitat-sim/releases) of Habitat-Sim? Your question may already be addressed in the latest version. We may also not be able to help with problems in earlier versions because they sometimes lack the more verbose logging needed for debugging. 
```

Comment:
Trailing whitespaces:
```suggestion
Habitat is under active development, and we advise users to restrict themselves to stable releases. Are you using the latest [release version](https://github.com/facebookresearch/habitat-sim/releases) of Habitat-Sim? Your question may already be addressed in the latest version. We may also not be able to help with problems in earlier versions because they sometimes lack the more verbose logging needed for debugging.
```